### ISSUE TITLE
P4 Test data

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/dlnd-issue-reports#253]
Just realized this.

The sample/test sentence provided to translate into French after training the model is from the training dataset itself.

Wouldn't it actually be better to have a very small test set (few sentences) to be able to test the model and how accurate it really is?

Isn't being able to correctly translate most or entirety of a sentence from the training dataset itself a sign of overfitting as well?

I am going through the helper code provided to understand better how you have divided the training and validation set, but I think a test set should be required instead of a sentence from the training dataset. 

Let me know what you think.

**Comment by Ortalarel** [comment]
Can you please add a link?

**Comment by sahiljuneja** [comment]
Link to what? The project?

https://github.com/udacity/deep-learning/tree/master/language-translation

**Comment by lcrucks** [comment]
P4 is being removed (replaced with new P5 for reinforcement learning), so this won't be fixed.
